api-client/src/models/__tests__/LocationModel.spec.ts

```diff
@@ -1,126 +1,173 @@
-import { PoiFeature } from '../../maps'
+import {PoiFeature} from '../../maps'
 import LocationModel from '../LocationModel'
 
 describe('LocationModel', () => {
-  describe('location', () => {
-    it('should return name if town, address and name is null', () => {
-      expect(
-        new LocationModel({
-          id: 1,
-          name: 'test',
-          country: null,
-          region: null,
-          state: null,
-          address: null,
-          town: null,
-          postcode: null
-        }).location
-      ).toBe('test')
+    describe('location', () => {
```

Comment:
Is this maybe a formatting Issue?

Reply:
Seems like it, yes. Running prettier should fix it.